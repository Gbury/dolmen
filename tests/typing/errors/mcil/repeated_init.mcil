(set-logic ALL)

(define-system S1
  :init true
  :init false 
)