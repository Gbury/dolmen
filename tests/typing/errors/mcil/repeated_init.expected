File "tests/typing/errors/mcil/repeated_init.mcil", line 5, character 2-13:
5 |   :init false 
      ^^^^^^^^^^^
Error :init attribute is not repeatable
